apps/qubit/modules/search/templates/_aggregation.php

```diff
@@ -26,7 +26,7 @@
         <li <?php if ($active) echo 'class="active"' ?>>
           <?php echo link_to(__(render_value_inline($bucket['display'])) . '<span>, ' . $bucket['doc_count'] . ' ' . __('results') . '</span>',
             array('page' => null, $name => $bucket['key'] == 'unique_language' ? null : $bucket['key']) +
-            $sf_data->getRaw('sf_request')->getParameterHolder()->getAll(), array('title' => '')) ?>
+            $sf_data->getRaw('sf_request')->getParameterHolder()->getAll(), array('title' => $bucket['display'])) ?>
```

Comment:
I _think_ it would be good to wrap the display text in the i18n function, e.g. `array('title' => __($bucket['display'])))`?

I don't know how the `$bucket['display']` value is set though, so I'm not 100% sure.

Reply:
D'oh. Good point. I've updated it.